checks/src/test/java/com/kozaxinan/android/checks/NetworkLayerClassJsonDetectorTest.kt

```diff
@@ -0,0 +1,674 @@
+package com.kozaxinan.android.checks
+
+import com.android.tools.lint.checks.infrastructure.TestFile
+import com.android.tools.lint.checks.infrastructure.TestFiles.bytes
+import com.android.tools.lint.checks.infrastructure.TestFiles.java
+import com.android.tools.lint.checks.infrastructure.TestFiles.kotlin
+import com.android.tools.lint.checks.infrastructure.TestLintTask.lint
+import com.kozaxinan.android.checks.NetworkLayerClassJsonDetector.Companion.ISSUE_NETWORK_LAYER_CLASS_JSON_CLASS_RULE
+import com.kozaxinan.android.checks.NetworkLayerClassJsonDetector.Companion.ISSUE_NETWORK_LAYER_CLASS_JSON_RULE
+import org.junit.Test
+
+private val ISSUES_TO_TEST = arrayOf(ISSUE_NETWORK_LAYER_CLASS_JSON_RULE, ISSUE_NETWORK_LAYER_CLASS_JSON_CLASS_RULE)
```

Comment:
nit: can you also update updates test for this notation?

Reply:
I don't understand 🤔

Reply:
Woow wrong word usaga.

Can you update OTHER TEST for thiis kind of private val ISSUE_TO_TEST. It is not super important.